badge-maker/package.json

```diff
@@ -26,7 +26,7 @@
     "badge": "lib/badge-cli.js"
   },
   "engines": {
-    "node": ">= 12",
+    "node": ">= 14",
```

Comment:
I always forget this, but what if any posture do we have relative to engine/dependency update impacts on the major version of our package itself? I.e. this doesn't require a major version bump of the package does it?

Reply:
Yep. The next package version will be 4.0.0. We decided that when we dropped testing on node 10. We've just not actually changed anything that matters since then so we've not done a 4.0.0 release yet.

https://github.com/badges/shields/pull/8351/files#diff-aea71380a717e7f6c60bfb50b18e320f64350f96a62fd3fb805d1aad351bca46